helm/0.1.0/README.md

```diff
@@ -0,0 +1,225 @@
+            - name: "evaluation"
+              triggeredAfter: "2m"
+              properties:
+                timeframe: "2m"
+
+```
+
+### Step 3: Add the Job-Executor configuration file
+
+The following job-executor configuration file (referred to as job config or `job/config.yaml`) allows you to
+* deploy using `helm upgrade`,
+* rollback using `helm rollback`, and
+* scale up as a remediation action using `kubectl`
+
+Add the following content to a file called *helm-job-config.yaml* in your current working directory:
+```yaml
+apiVersion: v2
+actions:
+  - name: "Deploy using helm"
+    events:
+      - name: "sh.keptn.event.deployment.triggered"
+    tasks:
+      - name: "Run helm"
+        files:
+          - /charts
+        env:
+          - name: IMAGE
+            value: "$.data.configurationChange.values.image"
+            valueFrom: event
+        image: "alpine/helm:3.7.2"
+        serviceAccount: "jes-deploy-using-helm"
+        cmd: ["helm"]
+        args: ["upgrade", "--create-namespace", "--install", "-n", "$(KEPTN_PROJECT)-$(KEPTN_STAGE)", "$(KEPTN_SERVICE)", "/keptn/charts/$(KEPTN_SERVICE).tgz", "--set", "image=$(IMAGE)", "--wait"]
+
+  - name: "Rollback using helm"
+    events:
+      - name: "sh.keptn.event.rollback.triggered"
+    tasks:
+      - name: "Run helm"
+        serviceAccount: "jes-deploy-using-helm"
+        image: "alpine/helm:3.7.2"
+        cmd: ["helm"]
+        args: ["rollback", "-n", "$(KEPTN_PROJECT)-$(KEPTN_STAGE)", "$(KEPTN_SERVICE)", "--wait"]
+
+  - name: "Scale using kubectl"
+    events:
+      - name: "sh.keptn.event.action.triggered"
+        jsonpath:
+          property: "$.data.action.action"
+          match: "scaling"
+    tasks:
+      - name: "Run kubectl"
+        serviceAccount: "jes-deploy-using-helm"
+        env:
+          - name: SCALING
+            value: $.data.action.value
+            valueFrom: event
+        image: "alpine/k8s:1.20.15"
+        cmd: ["sh"]
+        # Note: Hardcoded kubernetes namespace & KEPTN_SERVICE does most likely not match the deployment name
+        args: ["-c", "REPLICAS=$(kubectl -n ${KEPTN_PROJECT}-${KEPTN_STAGE} get deployment/${KEPTN_SERVICE} -o go-template='{{.spec.replicas}}');DESIRED=$((${SCALING}+${REPLICAS}));echo Scaling deployment to ${DESIRED} && kubectl -n ${KEPTN_PROJECT}-${KEPTN_STAGE} scale --replicas=${DESIRED} deployment/${KEPTN_SERVICE}"]
+```
+
+Add the job-executor configuration file to your Keptn service and stage:
+
+```bash
+PROJECT=podtato-head
+SERVICE=helloservice
+
+keptn add-resource --project=$PROJECT --service=$SERVICE --all-stages --resource=helm-job-config.yaml --resourceUri=job/config.yaml
+```
+
+**Accessing the Kubernetes API**
+
+By default, job-executor-service does not grant access to the Kubernetes api, the respective jobs (e.g., `helm install` or `helm upgrade`) would fail with `The connection to the server localhost:8080 was refused - did you specify the right host or port?`.
```

Comment:
```suggestion
By default, `job-executor-service` does not grant access to the Kubernetes API, so the respective jobs (e.g., `helm install` or `helm upgrade`) would fail with `The connection to the server localhost:8080 was refused - did you specify the right host or port?`.
```
It seems to me that you should either say "the Job Executor Service", "JES" (assumes you have defined the acronym somewhere at the top of the file), or you should put tick marks around job-executor-service.

Reply:
Good suggestion, I will rework the document!